src/components/WorkOrder/WorkOrderHeader.js

```diff
@@ -27,6 +28,15 @@ const WorkOrderHeader = ({
     return currentTime > appointmentStartTime
   }
 
+  const completionReason = () => {
```

Comment:
A small improvement here would be to just do the check for the special case of `workOrder.status === 'Work Complete'` and otherwise just return whatever is in `workOrder.status`. Then if more completion statuses are added to the system later this we don't need to worry about updating this function.